setup.py

```diff
@@ -62,6 +64,59 @@ def get_convolution_extensions():
 
     return [_convolve_ext]
 
+def get_include_path(library_name):
+    env_var_hint = os.getenv(f"{library_name.upper()}_INCLUDE_DIR")
+    if env_var_hint is not None and os.path.exists(env_var_hint):
+        return env_var_hint
+
+    possible_directories = [
+        "/xsimd/include/xsimd",
+        "/eigen3/eigen-3.4.0",
+        "/",
+        "/usr/include",
+        "/usr/local/include",
+        sysconfig.get_path('include'),
+    ]
+
+    for directory in possible_directories:
+        full_path = os.path.join(directory, library_name)
+        if os.path.exists(full_path):
+            return full_path
+
+    raise Exception(f"The {library_name} library was not found in any known directory.")
+
+def get_cpp_extensions():
+    cpp_transform_pkgdir = Path('hexrd') / 'transforms/cpp_sublibrary'
+    src_files = [str(cpp_transform_pkgdir / 'src/transforms.cpp'), str(cpp_transform_pkgdir / 'src/inverse_distortion.cpp')]
+
+    extra_compile_args = ['-O3', '-Wall', '-shared', '-std=c++20', '-funroll-loops']
```

Comment:
We should verify that C++20 is available on all of the versions of the compilers we want to support.

Reply:
Thanks for noting this. As mentioned in a previous comment, I'm happy to revert this to C++11.